backend/index.js

```diff
@@ -0,0 +1,44 @@
+const express    = require('express');
```

Comment:
What is the reason for changing from `server` to `backend`? Not that I'm opposed to it, I'm just curious if there is a deploy requirement of some sort?

Reply:
no exact reason. i felt that backend was a better umbrella term to catch everything going on back here, but now that i think about it server works just as well.